dotCMS/src/main/java/com/dotmarketing/db/DbType.java

```diff
@@ -0,0 +1,44 @@
+package com.dotmarketing.db;
+
+/**
+ * Enum for the Database Types available
+ * @author Andre Curione
+ */
+public enum DbType {
+
+    MYSQL("MySQL"),
+    POSTGRESQL("PostgreSQL"),
+    ORACLE("Oracle"),
+    MSSQL("Microsoft SQL Server"),
+    H2("H2");
+
+    private String dbType;
+
+    DbType(final String dbType) {
+        this.dbType = dbType;
+    }
+
+    public String getDbType() {
+        return dbType;
+    }
+
+    public static DbType getDbType(final String type) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this method 5, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1142)